app/models/consent_form.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: consent_forms
+#
+#  id          :bigint           not null, primary key
+#  recorded_at :datetime
+#  route       :integer          not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  session_id  :bigint           not null
+#
+# Indexes
+#
+#  index_consent_forms_on_session_id  (session_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (session_id => sessions.id)
+#
+
+class ConsentForm < ApplicationRecord
+  audited
+
+  belongs_to :session
+
+  enum :route, %i[website phone paper in_person self_consent], prefix: "via"
```

Comment:
These can't be `self_consent` or `in_person`, I think?

Reply:
Arguably `website` is the only valid option here.

Reply:
As discussed, `route` is removed.